javascript/javascript1/week1/Problem1/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Homework of week 1</title>
+    <link rel="stylesheet" href="./index.css">
+</head>
+<body>
+    <h1>Week1 problem 1</h1>
+    <script src="./week1prob1.js"></script>
```

Comment:
This `.js` file does not exist in the current folder, please fix this issue for other problems as well.

Reply:
Fixed the source file location problem and changed the name of the html file

Reply:
Please review it again